simulator/simple_planning_simulator/launch/simple_planning_simulator.launch.py

```diff
@@ -1,7 +1,21 @@
+# Copyright 2020 Tier IV, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
 from launch import LaunchContext
 from launch import LaunchDescription
 from launch.actions import DeclareLaunchArgument
-from launch.conditions import IfCondition
+# from launch.conditions import IfCondition
```

Comment:
Is there a reason why you commented this out rather than deleting the line?

Reply:
I've just commented it out as it may contain some intent of the original code author.
Should I erase it?

Reply:
(deleted)